app/api/v1/endpoints/bookings.py

```
@@ -23,23 +26,34 @@ def create_booking(
     """
-    # In a real application, you would:
     # 1. Find nearby available drivers
+    nearby_drivers = LocationService.find_nearby_drivers(db, booking_in.pickup_location)
     # 2. Calculate fare based on distance and time
+    fare = FareCalculator.calculate_fare(booking_in.pickup_location, booking_in.dropoff_location)
     # 3. Assign a driver
-    # For this example, we'll just create the booking
-    
+    driver = DriverService.assign_driver(db, booking_in, fare, nearby_drivers)
+    # For this example, we'll just create the bookin_in
+    surgePricing = LocationService.check_surge_pricing(db,booking_in.pickup_location,"yes");
```

Ruff (0.11.9)

Statement ends with an unnecessary semicolon

(E703)